src/ICT4Events/Views/MaterialSystem/Forms/MaterialSystem.cs

```diff
@@ -0,0 +1,383 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using SharedModels.Data.ContextInterfaces;
+using SharedModels.Data.OracleContexts;
+using SharedModels.Models;
+
+
+namespace ICT4Events.Views.MaterialSystem.Forms
+{
+    public partial class MaterialSystem : Form
+    {
+        #region Local Variables
+        private readonly Event _event;
+        private readonly IGuestContext _contextGuest;
+        private readonly IMaterialContext _contextMaterial;
+        private readonly IMaterialTypeContext _contextMaterialType;
+        private bool _rfidScanned;
+        #endregion
+
+
+        public MaterialSystem(Event ev)
+                {
+                    InitializeComponent();
+                    _event = ev;
+                    _contextGuest = new GuestOracleContext();
+                    _contextMaterial = new MaterialOracleContext();
+                    _contextMaterialType = new MaterialTypeOracleContext();
+
+                    //TODO Rfid Setting
+                    _rfidScanned = false;
+                }
+
+
+        #region Local Functions
+        //Checks wether a id already occurs in _materials
+        private bool UniqueID(int id)
+        {
+            var check = true;
+            foreach (var material in _contextMaterial.GetAllByEvent(_event))
+            {
+                if (material.ID == id)
+                {
+                    check = false;
+                }
+            }
+            if (!check)
+                MessageBox.Show("Artikelnummer zit al in de lijst van materialen");
+            return check;
+        }
+        
+        //Checks if fields is/are filled
+        private bool FieldsFilled(List<TextBox> TextBoxes)
+        {
+            var Check = true;
+            foreach (var tb in TextBoxes)
+            {
+                if (string.IsNullOrEmpty(tb.Text))
+                    Check = false;
+            }
+            return Check;
+        }
+
+        private void UpdateArtNummerArtNaam()
+        {
```

Comment:
Nederlandse naamgevingen, locals met hoofdletters, controls die anders genoemd zijn dan in alle andere systemen, etc
Veel van de naamgevingen in deze commit voldoen niet aan de eisen die gedefinieerd zijn binnen de coding standards.